### PR TITLE
fix: Set proper time cov in TrackParamsEstimationAlgorithm

### PR DESCRIPTION
**Pull request by andiwand**
The time covariance we currently set in `TrackParamsEstimationAlgorithm` causes numerical instabilities. In this PR I set something more reasonable which fixes downstream issues for the vertexing

Besides that I linked the subloggers in the CKF properly and removed the regularization term from the KF filter and smoothing as this seems to be stable now

### CODE REVIEW

File: Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp [hunk 1]
```diff
@@ -950,7 +950,7 @@ class CombinatorialKalmanFilter {
         } else {
           // Kalman update
           auto updateRes = m_extensions.updater(
-              gctx, trackState, Direction::Forward, getDummyLogger());
+              gctx, trackState, Direction::Forward, *logger().clone("updater"));
```
Comment (andiwand) [inline Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp:953]:
```suggestion
              gctx, trackState, Direction::Forward, *logger().cloneWithSuffix("updater"));
```

File: Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp [hunk 2]
```diff
@@ -1129,9 +1129,9 @@ class CombinatorialKalmanFilter {
       ACTS_VERBOSE("Apply smoothing on " << nStates
                                          << " filtered track states.");
       // Smooth the track states
-      auto smoothRes =
-          m_extensions.smoother(state.geoContext, *result.fittedStates,
-                                lastMeasurementIndex, getDummyLogger());
+      auto smoothRes = m_extensions.smoother(
+          state.geoContext, *result.fittedStates, lastMeasurementIndex,
+          *logger().clone("smoother"));
```
Comment (andiwand) [inline Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp:1134]:
```suggestion
          *logger().cloneWithSuffix("smoother"));
```

**Comment by andiwand** [comment]
this gets hammered by FPEs again :(

temporarily disabled them to check the physmon performance

File: CI/physmon/physmon.py [hunk 3]
```diff
@@ -366,8 +366,9 @@ def run_vertexing(fitter, mu, events):
         )
 
 
-with acts.FpeMonitor():
+import contextlib
 
+with contextlib.nullcontext():
```
Comment (andiwand) [inline CI/physmon/physmon.py:371]: revert

**Comment by andiwand** [comment]
closing after integration in https://github.com/acts-project/acts/pull/2145/

**Comment by andiwand** [comment]
this works now :tada:

File: Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackParamsEstimationAlgorithm.hpp [hunk 4]
```diff
@@ -65,7 +65,7 @@ class TrackParamsEstimationAlgorithm final : public IAlgorithm {
     /// q/p resolution.
     double sigmaQOverP = 0.1 / Acts::UnitConstants::GeV;
     /// Time resolution.
-    double sigmaT0 = 1400 * Acts::UnitConstants::s;
+    double sigmaT0 = 10 * Acts::UnitConstants::ns;
```
Comment (paulgessinger) [inline Examples/Algorithms/TrackFinding/include/ActsExamples/TrackFinding/TrackParamsEstimationAlgorithm.hpp:68]: So this is arbitrary but smaller? Should we make this configurable?

File: Core/src/TrackFitting/GainMatrixUpdater.cpp [hunk 5]
```diff
@@ -51,8 +51,6 @@ std::tuple<double, std::error_code> GainMatrixUpdater::visitMeasurement(
                         .inverse())
                        .eval();
 
-    ACTS_VERBOSE("Gain Matrix K:\n" << K);
```
[the remark this answers is not in the document]
Comment (andiwand) [reply Core/src/TrackFitting/GainMatrixUpdater.cpp:54]: I think that was just a mistake when I cleaned up my own debug prints - reverting

**Review by benjaminhuth — APPROVED** [review]
looks good!

**Comment by benjaminhuth** [comment]
But still some hash problems @andiwand

**Comment by andiwand** [comment]
had to update it again due to some conflicts

**Review by benjaminhuth — APPROVED** [review]

**Pull request merged**